wrappers/winnt.go

```diff
@@ -1209,6 +1209,16 @@ const (
 	JobObjectGroupInformation                   = 11
 )
 
+const (
+	EVENT_ALL_ACCESS   = 0x1F0003
+	EVENT_MODIFY_STATE = 0x0002
+)
+
+const (
+	MUTEX_ALL_ACCESS   = 0x1F0001
+	MUTEX_MODIFY_STATE = 0x0001
```

Comment:
The constants defined in winnt.h are `MUTANT_MODIFY_STATE` and `MUTANT_ALL_ACCESS`.  Then, `MUTEX_MODIFY_STATE` and `MUTEX_ALL_ACCESS` are defined in terms of them in winbase.h.